### DISCUSSION TITLE
The Allegro API interface is unable to retrieve the shipmentId via orderId. We hope a new interface can be provided.

### DISCUSSION THREAD

**Discussion by aron007** [allegro/allegro-api#9932] [Suggestions - rejected]
The Allegro API interface is unable to retrieve the shipmentId via orderId. We hope a new interface can be provided.

![1726767481420](https://github.com/user-attachments/assets/725cd7aa-eb8e-4769-8d07-c24f1e6df481)

**Comment by MartaNowaczyk** [comment]
Thank you for your suggestion. I have forwarded it to the team responsible for this endpoint.

**Comment by jezewskiBNetrox** [comment]
@aron007 I'd like to mention that is frontend API meant to be used by Allegro website, not API Allegro this GitHub repo is about.

**Reply by aron007** [reply]
Yes, Allegro API does not have a similar interface, it cannot obtain the shipmentId through orderId
